### DISCUSSION TITLE
Add support for XIAO nRF52840 (sense)?

### DISCUSSION THREAD

**Discussion by Tjoms99** [h2zero/n-able-Arduino#23] [Q&A]
I see there are a lot of nRF52840 boards supported, but not the XIAO nRF52840. Would it be possible to add it as a supported board?

**Comment by h2zero** [comment]
Yes it would be possible for sure. If you know the pins etc.. then an new board can be added just as the others are.

**Reply by babolcs** [reply]
Thanks for your work in advance. I try to use this board as well with the NimBLE.
